code/src/components/Header.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+const Header = () => {
```

Comment:
I would consider this component for a bit and the chosen element that you return here. In general, an article tag is usually intended for forum posts, newspaper articles, blog entries etc. Since you’re only displaying a heading and a span in this component, my suggestion would be to wrap this in a div tag. Another suggestion is to wrap it in a header tag, which you in App.js would have to mount outside of the main element that you currently have wrapping the PostMessage and ListMessage components.

Also, personally I would’ve expected to see a header element based on the naming of the component, but that's just me maybe 🙂